wordpress-mobile/gutenberg-mobile
# Use gutenberg translation files locally

Addresses part of #203 
Requires https://github.com/wordpress-mobile/WordPress-Android/pull/9249
Requires https://github.com/wordpress-mobile/WordPress-iOS/pull/11056

~~Requires those 2 PRs to be merged first so the gb-mobile strings get translated in WPiOS and WPAndroid glotpress:
https://github.com/wordpress-mobile/WordPress-iOS/pull/10999
https://github.com/wordpress-mobile/WordPress-Android/pull/9214~~

### Summary
This PR adds a local script that downloads gutenberg translation files and cache them locally.

### Testing Instructions
- Load this branch locally
- Run `yarn start` and check that translation files are downloaded in `i18n-cache/data` and that a file is generated at `i18n-cache/index.native.js`
- Edit `src/app/App.js` and replace `setLocaleData( getTranslation( locale ) );` with `setLocaleData( getTranslation( 'es' ) );`
- Reload the app and check that parts of the app have been translated to Spanish

- Reset the code and run `yarn wpandroid` or `yarn ios`
- Change the app or the phone language
- Check that gutenberg editor is translated

A PR in both https://github.com/wordpress-mobile/WordPress-iOS and https://github.com/wordpress-mobile/WordPress-Android will be necessary to pass the new `locale` parameter.
